hubmap/frontend/cypress/integration/hubmap_spec.js

```diff
@@ -57,7 +57,8 @@ describe('HuBMAP', () => {
     cy.visit('/');
 
     cy.contains('Browse').click();
-    cy.contains('Data Analysis').click();
+    // https://github.com/rakannimer/react-google-charts/blob/master/cypress/integration/charts.spec.js
```

Comment:
I think if we include links, we should explain what they are about. In this case, though, I think it's clear enough what we're doing without any link:
```suggestion
    // Google charts are very slow to render, so increase timeout.
```

But:
- Maybe this should just be made a global setting, if we're going to need more time on Travis?
- Maybe (in the long term) we should have second thoughts about libraries that  are so heavy-weight that they need this much time.